lib/src/rules/implicit_reopen.dart

```diff
@@ -87,35 +87,30 @@ class _Visitor extends SimpleAstVisitor {
 
   void checkElement(InterfaceElement? element, NamedCompilationUnitMember node,
       {required String type}) {
-    if (element == null) return;
+    if (element is! ClassElement) return;
```

Comment:
Is it impossible for a mixin to re-open another mixin (or a `mixin class`)? (That might be true, I'm just not sure.)

Reply:
I'm not sure either, but this lint rule only registers `ClassDeclaration`, so `element` _should_ only be a ClassDeclaration. I'd like to investigate this mixin question though.

Reply:
Drive-by comment: It is not possible to reopen a mixin or a mixin class: It has the modifier `base` or no modifier, and no subtype of a `base` declaration can exist without a modifier which is at least as strong as `base`.